SLHCUpgradeSimulations/Geometry/test/phase2_reco_pixelntuple_cfg.py

```diff
@@ -2,39 +2,40 @@
 # using: 
 # Revision: 1.19 
 # Source: /local/reps/CMSSW/CMSSW/Configuration/Applications/python/ConfigBuilder.py,v 
-# with command line options: step2 --conditions auto:phase2_realistic -s DIGI:pdigi_valid,L1,L1TrackTrigger,DIGI2RAW,HLT:@fake2,RAW2DIGI,L1Reco,RECO --datatier GEN-SIM-RECO -n 10 --geometry Extended2026D41 --era Phase2 --eventcontent FEVTDEBUGHLT --filein file:SingleMuPt1000_pythia8_cfi_GEN_SIM.root --runUnscheduled --no_exec
+# with command line options: step3 --conditions auto:phase2_realistic_T21 -s RAW2DIGI,L1Reco,RECO,RECOSIM,PAT,VALIDATION:@phase2Validation+@miniAODValidation,DQM:@phase2+@miniAODDQM --datatier GEN-SIM-RECO,MINIAODSIM,DQMIO -n 10 --geometry Extended2026D66 --era Phase2C11 --eventcontent FEVTDEBUGHLT,MINIAODSIM,DQM --filein file:step2.root --fileout file:step3.root
 import FWCore.ParameterSet.Config as cms
 
 
-from Configuration.Eras.Era_Phase2C9_cff import Phase2C9
-process = cms.Process('Phase2PixelNtuple',Phase2C9)
+from Configuration.Eras.Era_Phase2C11_cff import Phase2C11
+process = cms.Process('USER',Phase2C11)
 
 # import of standard configurations
 process.load('Configuration.StandardSequences.Services_cff')
 process.load('SimGeneral.HepPDTESSource.pythiapdt_cfi')
 process.load('FWCore.MessageService.MessageLogger_cfi')
 process.load('Configuration.EventContent.EventContent_cff')
 process.load('SimGeneral.MixingModule.mixNoPU_cfi')
-process.load('Configuration.Geometry.GeometryExtended2026D49Reco_cff')
+process.load('Configuration.Geometry.GeometryExtended2026D66Reco_cff')
```

Comment:
is it intentional that the reco test config uses a different geometry than the digi-reco config above?

Reply:
Yes they both use the correct geometry for their GT and the digi-reco one has to match the relval sample. I can change the reco test config to the T15 geometry if its desired.